bitnami/harbor/README.md

```diff
@@ -210,6 +212,7 @@ The following table lists the configurable parameters of the Harbor chart and th
 | `chartMuseumImage.debug`                                                    | Specify if debug logs should be enabled                                  | `false`                                                 |
 | `chartmuseum.enabled`                                                       | Enable ChartMuseum                                                       | `true`                                                  |
 | `chartmuseum.replicas`                                                      | Number of ChartMuseum replicas                                           | `1`                                                     |
+| `chartmuseum.updateStrategy`                                                 | The update strategy for deployments with persistent volumes: RollingUpdate or Recreate. Set it as Recreate when RWM for volumes isn't supported | `RollingUpdate`                                                     |
```

Comment:
This should be `chartmuseum.updateStrategy.type`. This applies to the rest of services too.